pkg/kubelet/cm/cpumanager/state/state_checkpoint.go

```diff
@@ -54,10 +55,21 @@ func NewCheckpointState(stateDir, checkpointName, policyName string, initialCont
 	}
 
 	if err := stateCheckpoint.restoreState(); err != nil {
-		//lint:ignore ST1005 user-facing error message
-		return nil, fmt.Errorf("could not restore state from checkpoint: %v\n"+
-			"Please drain this node and delete the CPU manager checkpoint file %q before restarting Kubelet.",
-			err, path.Join(stateDir, checkpointName))
+		klog.Errorf("could not restore state from checkpoint: %v", err)
+		checkpointFile := path.Join(stateDir, checkpointName)
+		corruptpath := checkpointFile + ".corrupt"
+		if err := os.Rename(checkpointFile, corruptpath); err != nil {
```

Comment:
if a kubelet really did encounter this scenario, I'm not sure what the consequences are of discarding the invalid file and allowing the kubelet to continue without draining current pods.